scriptTag/css/script.css

```diff
@@ -0,0 +1,3 @@
+function fun2(){
```

Comment:
# File specific comment

please change extension of file from **.css** to **.js**

Reply:
done